### ISSUE TITLE
Random input for SubscriptionId is a string, not a guid

### ISSUE THREAD

**Issue by abatishchev** [Azure/oav#792]
Common Types v3 unfortunately defines the `SubscriptionIdParameter` as just a string of min length 1:

https://github.com/Azure/azure-rest-api-specs/blob/77b673c60d3efe1bc9cb30525db176e67b8d24dd/specification/common-types/resource-management/v3/types.json#L622-L629

So sample input is just a string too:

```json
{
  "parameters": {
    "api-version": "2022-06-01-preview",
    "subscriptionId": "aaaaaaaaaaaaaa",
    "resourceGroupName": "rgthrottling"
  }
}
```
But `oav` could be smarted and generate a guid instead.

**Comment by raych1** [comment]
Mock values are based on the type and format. Sub id is an exceptional case to be filled with guid.

**Comment by abatishchev** [comment]
> sub id is an exceptional case to be filled with guid

Yes, I think that would be an elegant solution that balances out addressing the issue and minimizing the potential blast radius.

**Comment by raych1** [comment]
subId will be filled with GUID and the change will be released at V3.2.1
